RELEASE.md

```diff
@@ -238,6 +238,30 @@ Create a new release for the newly created Git tag: https://github.com/Icinga/ic
 
 Use the changelog for the release body.
 
+## Docker
+
+> Only for final versions (not for RCs).
+
+Once the release has been published on GitHub, wait for its
+[GitHub actions](https://github.com/Icinga/icingaweb2/actions) to complete.
+
+```bash
+VERSION=2.9.1
+
+TAGS=(2.9)
+#TAGS=(2.9 2 latest)
```

Comment:
What's the `2` for? And why is the tag still `2.9` although it's the `2.9.1` we're pulling?

Reply:
In the commented-out case 2.9.1 is the lastest 2.9.x, the lastest 2.x.y and the lastest in general.

Reply:
So there won't be a tag for a specific minor version? Why that?

Reply:
It will be there once the release GHA are done.